web/src/components/HeaderLanguageSelectorItem.tsx

```diff
@@ -5,29 +5,52 @@ import languageIcon from '../assets/language-icon.svg'
 import SelectorItemModel from '../models/SelectorItemModel'
 import HeaderActionItemDropDown from './HeaderActionItemDropDown'
 import HeaderActionBarItemLink from './HeaderActionItemLink'
+import KebabActionItemDropDown from './KebabActionItemDropDown'
 import Selector from './Selector'
 
 type PropsType = {
   selectorItems: Array<SelectorItemModel>
   activeItemCode: string
   t: TFunction<'layout'>
+  inKebabMenu?: boolean
 }
 
-const HeaderLanguageSelectorItem = ({ selectorItems, activeItemCode, t }: PropsType): ReactElement => {
+const HeaderLanguageSelectorItem = ({
+  selectorItems,
+  activeItemCode,
+  t,
+  inKebabMenu = false
+}: PropsType): ReactElement => {
   const noLanguagesHint = t('noLanguages')
 
+  const closeDropDown = (): ReactElement => (
+    <Selector
+      closeDropDown={closeDropDown}
+      verticalLayout={false}
+      items={selectorItems}
+      activeItemCode={activeItemCode}
+      disabledItemTooltip={t('noTranslation')}
+    />
+  )
```

Comment:
Something looks wrong here: You define a react component which receives itself as `closeDropDown` prop? Is this an error or could you explain this?

Edit: Ok, I think it was bad code before but now you are not passing the `closeDropDown` anymore as prop. It probably should be if we want to keep the previous way:
```suggestion
  const Item = (closeDropDown: () => void): ReactElement => (
    <Selector
      closeDropDown={closeDropDown}
      verticalLayout={false}
      items={selectorItems}
      activeItemCode={activeItemCode}
      disabledItemTooltip={t('noTranslation')}
    />
  )
```